docs/faq.md

```diff
@@ -101,12 +101,14 @@ Segmentation faults and similar errors on macOS
 
 __Solution__
 
+
```

Comment:
Looks like some excess whitespace added. Does this link shortcut resolve to the issue?

Reply:
Updated